nixos/tests/systemd.nix

```diff
@@ -69,5 +77,20 @@ import ./make-test.nix {
       # has a last mount time, because the file system wasn't checked.
       $machine->fail('dumpe2fs /dev/vdb | grep -q "^Last mount time: *n/a"');
     };
+
+    # Regression test for https://github.com/NixOS/nixpkgs/issues/35268
+    subtest "file system with x-initrd.mount is not unmounted", sub {
+      $machine->shutdown;
+      $machine->waitForUnit('multi-user.target');
+      # If the file system was unmounted during the shutdown the file system
+      # has a last mount time, because the file system wasn't checked.
+      $machine->fail('dumpe2fs /dev/vdb | grep -q "^Last mount time: *n/a"');
+    };
```

Comment:
This seems to have been duplicated from above. Reverted in https://github.com/NixOS/nixpkgs/pull/66825.

Reply:
Thanks!